### PR TITLE
Elixir version

### PR DESCRIPTION
**Pull request by franzejr**

### CODE REVIEW

File: elixir/maybe_hug.ex [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import IO
+require Cutie
+
+cutie_person = %{:name => "Jonh",:accepts => true}
```
Comment (zkat) [inline elixir/maybe_hug.ex]: `s/Jonh/John/`?

File: elixir/maybe_hug.ex [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+not_cutie_person = %{:name => "Mary",:accepts => false}
+
+puts Cutie.hug(cutie_person)
+puts Cutie.hug(not_cutie_person)
```
Comment (zkat) [inline elixir/maybe_hug.ex]: `s/not_cutie_person/empathy_person/` and `s/cutie_person/huggable_person/`

We make no judgment about whether they're cuties or not -- they are always cuties. The question is whether to hug them or empathise!

File: elixir/maybe_hug.ex (hunk 1 above)
Comment (franzejr) [inline elixir/maybe_hug.ex]: Yeah! I've added a name in the Cutie. What do you think ? =D

File: elixir/maybe_hug.ex [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import IO
+require Cutie
+
+cutie_person = %{:name => "Jonh",:accepts => true}
+empathic_person = %{:name => "Mary",:accepts => false}
```
Comment (franzejr) [inline elixir/maybe_hug.ex:5]: @zkat what do you think?

Comment (zkat) [inline elixir/maybe_hug.ex:5]: :+1:

File: elixir/maybe_hug.ex (hunk 1 above)
Comment (zkat) [inline elixir/maybe_hug.ex]: I mean, you misspelled "John" :)

**Comment by franzejr** [comment]
@zkat what do you think now ?

**Comment by zkat** [comment]
Looks good!

``` elixir
puts Cutie.hug(@franzejr)
```

**Pull request merged**